samples/nrf9160/mqtt_simple_only_secure/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(hello_world)
+
+zephyr_linker_sources(SECTIONS app-data-section.ld)
+target_sources(app PRIVATE src/main.c)
```

Comment:
Instead of copying the entire source dir, could you simply point to the source in the original sample folder, and then have the separate prj.conf in this folder as well. Assuming that there are no code chanes.

Reply:
The sample file is mostly for testing and just ensuring that it actually works I think I would remove it in favor of document how to build your application. So by using the `__in_section` we should be able to place whatever we prefer in the split RAM. Using https://docs.zephyrproject.org/latest/guides/code-relocation.html will actually allow you to place .bss/.text/.data sections in the split RAM also. So if you fill up your first 64K you should be able to place additional sections into `SRAM_AFTER_BSDLIB` 128K